### DISCUSSION TITLE
스크럼 시간 변경

### DISCUSSION THREAD

**Discussion by JiinSeok** [team-final/albaform#74] [General]
대학생 @gayeong0916 님의 시간표 상황 상
금요일에는 3시 팀미팅에 참석이 어렵다고 합니다. (끝나면 5시~5시반)

변경안
- [x] 저녁식사 후 8시로 시간 변경 (마무리 느낌)
- [x] PR 리뷰 하고 참석
- [ ] 아침 동기부여를 위한 모닝멘션 🥹

**Comment by JuhyeokC** [comment]
금요일은 저녁 8시로 하고 그 전에 PR 리뷰 하기!

**Reply by JiinSeok** [reply]
좋아요! 일단 이거는 확정

**Comment by JiinSeok** [comment]
카톡으로 더 자세한 이야기!!!!!
